src/content/docs/design/customize-with-code/preview-custom-design.mdx

```diff
@@ -0,0 +1,28 @@
+---
+page_id: a56552a5-e4eb-4c71-b44a-0070813de59f
+title: Preview custom code
+sidebar:
+  order: 14
+relatedArticles:
+  - 66f83c18-d261-48b8-a517-648ed1b7b064
+  - b42663d8-5e2a-42cd-859a-f9737a1495df
+  - 743d264d-ef08-4f2e-a42c-2b971152dccf
+  - 94a22da4-2013-4253-99dc-d686861cc320
+---
+
+<Aside type="upgrade">
+
+You can only preview custom code if you are on a Kinde Plus or Scale plan.
+
+</Aside>
+
+Unlike changes you make in the Kinde UI that impact page layouts and content, your custom designs can’t be viewed in a Kinde-generated preview. 
+
+Instead, you can test designs using your real auth flow, using preview credentials. 
+
+## Preview custom code
+
+1. Go to **Design > Custom code > Preview**. 
+2. Copy the preview credentials, including **Test ID** and **Password** to use in an authentication request.
+
```

Comment:
@clairekinde11 
1. In your application code, you will need to pass the additional auth url parameter `pages_mode=preview`. 
2. When the auth flow is initiated a password screen will appear which is required to view the preview
3. Enter your preview password (this can found inside Kinde at **Design > Custom code > Preview)